types/src/on_chain_config/diem_version.rs

```diff
@@ -20,5 +20,9 @@ impl OnChainConfig for DiemVersion {
 // TODO: expand the list if more features are gated
 pub const DIEM_VERSION_2: DiemVersion = DiemVersion { major: 2 };
 
+// Version at which multi-agent txn feature is activated.
+// TODO: use a feature flag instead when it becomes available
+pub const DIEM_VERSION_MULTI_AGENT: DiemVersion = DiemVersion { major: 10 };
```

Comment:
What is benefit to gate it as version 10 instead of 3? 
should it be 3? or DIEM_VERSION_3? current versioning can't gate 2 different features with different numbers.
if there is new feature X gated at 4, and when we need to enable multi agent, but the feature X has problem and can't be turned on, what we do?

Reply:
also, notice DIEM_MAX_KNOWN_VERSION is set to DIEM_VERSION_2, do we need update it to match DIEM_VERSION_MULTI_AGENT?

Reply:
After talking offline, we have decided to use DIEM_VERSION_3.